portfolio/src/main/webapp/js/script.js

```diff
@@ -76,8 +76,19 @@ async function getNumComments(numComments) {
   const messages = await response.json();
   const messageContainer = document.getElementById('message-container');
   messageContainer.innerHTML = '';
-  for (const message of messages) { 
-    messageContainer.appendChild(createListElement(message));
+  document.getElementById('login').setAttribute('href', messages.url);
+  if (JSON.parse(messages.loggedin)) { 
+    document.getElementById('login').innerHTML = 'Logout';
+    for (const message of JSON.parse(messages.comments)) { 
+      if (messages.user == message.useremail) {
+        messageContainer.appendChild(createListElement(message, true));
```

Comment:
what about:

```
messageContainer.appendChild(
  createListElement(message, /*deleteable=*/messages.user == messages.useremail));
```

Note that parameter name comments when it's unclear what parameter it is just by reading the call site here: https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments